alembic/versions/b74cc308b1ec_move_responses_to_db.py

```diff
@@ -0,0 +1,83 @@
+revision = 'b74cc308b1ec'
+down_revision = '66d78d1a29ee'
+branch_labels = None
+depends_on = None
+
+import alembic
+import sqlalchemy
+import json
+from collections import defaultdict
+from www.commands import build_response_dict
+
+def upgrade():
+	commands = alembic.op.create_table("commands",
+		sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True, autoincrement=True),
```

Comment:
`autoincrement=True` uses `SERIAL` and [it's recommended to use identity columns instead](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial).

```suggestion
		sqlalchemy.Column("id", sqlalchemy.Integer, sqlalchemy.Identity(), primary_key=True),
```